functions/tips/lazy/response

```diff
@@ -0,0 +1,2 @@
+
+instance: 2021-02-20 00:03:03 +0000; function: 2021-02-20 00:03:07 +0000
```

Comment:
This probably needs updating. The current output is `"Lazy: ... non_lazy: ..."` and also it outputs integer computations results rather than timestamps.